src/Our.Umbraco.DocTypeGridEditor/Web/PreviewModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Net.Http.Formatting;
+using Umbraco.Core.Models;
+
+namespace Our.Umbraco.DocTypeGridEditor.Web
+{
+    public class PreviewModel
+    {
+        public IPublishedContent Page { get; set; }
+
+        public FormDataCollection Values { get; set; }
```

Comment:
I'm not sure about exposing `FormDataCollection` from the preview model.  It's not a showstopper, but thinking that we already know what the properties are, (e.g. "editorAlias", "contentTypeAlias", etc), then we could strongly-type them? (since we're already introducing a new view-model for the previewer.

Like I say, it's not a showstopper for this PR ... just something for me & @mattbrailsford to discuss.